### PR TITLE
update SRIs

### PR DESCRIPTION
**Pull request by glasserc**
r? @n1k0

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -40,5 +40,5 @@ integrity with the hash provided below:
 | Filename                | Hash                                                                    |
 |-------------------------|-------------------------------------------------------------------------|
 | kinto.js                | sha384-M+I8zk+At4bXKx0yVyRz/j05JlFeR8+wLbvHQPsdW5dixtXywViPOT3/ueqt1BuV |
```
Comment (leplatrem) [inline docs/installation.md]: full bundle was not changed ?

**Comment by n1k0** [comment]
LGTM with kinto.js SRI updated.

File: docs/installation.md (hunk 1 above)
Comment (glasserc) [inline docs/installation.md]: Right. The only changes this release were `fx-src/index.js` and dependencies.

**Comment by leplatrem** [comment]
r+

**Pull request merged**